src/utils/constants.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum, auto
+
+
+class ChoicesEnum(Enum):
+    """Augments a regular enum with methods to return lists of enum
+    choices/names."""
+
+    @classmethod
+    def get_names(cls):
+        names = []
+        for choice in cls:
+            names.append(choice.name)
+
+        return names
+
+    @classmethod
+    def get_choices(cls):
+        choices = []
+        for choice in cls:
+            choices.append(choice)
+
+        return choices
+
+
+class Requests(ChoicesEnum):
+    START = auto()
+    STARTING = auto()
+    STOP = auto()
+    STOPPING = auto()
+    ARE_YOU_ALIVE = auto()
+    YES = auto()
+
+
+class RunLevels(ChoicesEnum):
+    LOCAL = auto()
+    LOCAL_STANDALONE = auto()
+    PRODUCTION = auto()
+
+
+class LogLevels(ChoicesEnum):
+    DEBUG = auto()
+    INFO = auto()
+    WARNING = auto()
+    ERROR = auto()
+
+
+class ActorNames(ChoicesEnum):
```

Comment:
more specific name for this enum would be preffered. I.e "BellboyActors"

Reply:
Sure? Not certain that your proposed name is more specific; this enum does contain the actor names.

Reply:
BellboyActors is more specific and direct imo .. nbd